packages/backend/src/misc/is-not-null.ts

```diff
@@ -0,0 +1,3 @@
+export function isNotNull<T>(input: T | undefined | null): input is T {
```

Comment:
生やすだけ生やして使ってない？
あと

```suggestion
export function isNotNull<T extends {}>(input: T | undefined | null): input is T {
```

であるべき

Reply:
使ってる
https://github.com/misskey-dev/misskey/pull/10133/files#diff-7546b9350a0555e1aeb3fbdbcb8fc2d8c8da90a0149d4a3e741bb2068bcba3daL307
後半についてはたしかに

Reply:
> @typescript-eslint/ban-types

obsolete だなこれ